### PR TITLE
Add note to desc's on how to determine GPIO pin numbers

### PR DESCRIPTION
**Pull request by Hwurzburg**
@rmackay9 per our discussion....the GPIOs page shows how to look up GPIO pin numbers pretty well, I think...

### CODE REVIEW

File: libraries/AP_RPM/AP_RPM_Params.cpp [hunk 1]
```diff
@@ -20,11 +20,11 @@ const AP_Param::GroupInfo AP_RPM_Params::var_info[] = {
     // @Param: TYPE
     // @DisplayName: RPM type
     // @Description: What type of RPM sensor is connected
-    // @Values: 0:None,1:Not Used,2:AUXPIN,3:EFI,4:Harmonic Notch,5:ESC Telemetry Motors Bitmask
+    // @Values: 0:None,1:Not Used,2:GPIO,3:EFI,4:Harmonic Notch,5:ESC Telemetry Motors Bitmask
```
Comment (rmackay9) [inline libraries/AP_RPM/AP_RPM_Params.cpp:23]: I agree that AUXPIN was not a good description.  this change is a good one I think.

Comment (rmackay9) [reply libraries/AP_RPM/AP_RPM_Params.cpp:23]: A little lower in this file (lin 59) it says, "Only used on type = PWM." which is out-of-date even for what we have in master.  Perhaps this lower down comment should also be updated to mention "GPIO"

**Review by rmackay9 — APPROVED** [review]
Looks like an improvement to me.  I've made a suggestion on updating another comment but it is an existing issue in any case.

File: libraries/AP_RPM/AP_RPM_Params.cpp (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_RPM/AP_RPM_Params.cpp:23]: fixed

**Comment by rmackay9** [comment]
I've merged this to save us some time on the dev call.  Thanks!

**Pull request merged**